rhcephcompose/compose.py

```diff
@@ -72,8 +73,10 @@ def validate(self):
         """
         Sanity-check that files exist before we go to the work of running.
         """
-        # builds lists
-        self.validate_builds_lists()
+        if not self.koji_profile:
+            if self.chacra_url is None:
+                raise RuntimeError('Set koji_profile or chacra_url')
+            self.validate_builds_lists()
```

Comment:
if I am reading this correctly, this should only be done when `self.chacra_url` is configured right? never when `self.koji_profile` is?

Reply:
That's correct. It would be clearer if "validate_builds_list()" was named "validate_chacra_builds_list()"